.gitignore

```diff
@@ -12,3 +12,4 @@ __pycache__
 *.iml
 /venv
 /out
+tools/jarcat/zip/*.zip
```

Comment:
What was creating this?

Reply:
jarcat tests :)

Reply:
They create them in the src root? That's not good.

Reply:
yes, somehow I got the zips there. Let me try and reproduce.